File: tests/fixtures/eval_go/pkg/format/format.go
package format

func FormatMoney(amount int, currency string) string {
	return currency + " " + string(rune(amount))
}

func UpperFirst(text string) string {
	if text == "" {
		return text
	}
	return string(rune(text[0]-32)) + text[1:]
}

File: tests/fixtures/eval_rust/src/format.rs
pub fn format_money(amount: u32, currency: &str) -> String {
    format!("{}{}", currency, amount)
}

pub fn upper_first(text: &str) -> String {
    let mut out = String::new();
    let mut first = true;
    for ch in text.chars() {
        if first {
            out.extend(ch.to_uppercase());
            first = false;
        } else {
            out.push(ch);
        }
    }
    out
}
